addons/website/views/website_templates.xml

```diff
@@ -330,6 +330,28 @@
     </xpath>
 </template>
 
+<!-- Navbar templates -->
+<template id="template_navbar_showcase" inherit_id="website.layout" name="Template Navbar Showcase" active="False">
+    <xpath expr="//header//nav[hasclass('navbar')]" position="attributes">
+        <attribute name="class" add="o_navbar_showcase" separator=" "/>
+        <attribute name="class" remove="navbar-expand-md" separator=" "/>
+    </xpath>
+    <xpath expr="//header//button[hasclass('navbar-toggler')]" position="attributes">
+        <attribute name="class" add="order-first" separator=" "/>
+    </xpath>
```

Comment:
Would not it be better to actually move the toggler at the correct position in the DOM instead of doing that with a class ? (with a xpath position="move")